lib/ext/adviser.rb

```diff
@@ -0,0 +1,19 @@
+require 'csv'
+
+module AdviserExtension
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def to_csv(_ = {})
+      CSV.generate do |csv|
+        csv << ['Ref. Number', 'Name', 'Firm']
+
+        find_each do |adviser|
+          csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+        end
+      end
+    end
+  end
+end
+
+ActiveRecord::Base.send(:include, AdviserExtension)
```

Comment:
Agree with @tomas-stefano, the test - [here](https://github.com/moneyadviceservice/rad/pull/406/files#diff-fcb272b2a5bcd68e29fa8e3e4051789dR1) should probably be updated to more explicitly test this code